src/styles/pages/_coming-soon.scss

```diff
@@ -98,6 +98,11 @@
             padding: rem(10);
         }
 
+        .invalid
+        {
+            border: 1px solid #ff002f;
```

Comment:
FYI: there's a custom sass function rem() whose input is the px you want and it will return its rem equivalient